src/Enum/IssueStatusEnum.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Enum;
+
+/*
+Structure grabbed from Leantime:
+https://github.com/Leantime/leantime/blob/f81a5cbac4377f43719f77f909c79c15a0113f0a/app/Domain/Tickets/Repositories/Tickets.php#L48
+*/
+
+enum IssueStatusEnum: string
```

Comment:
We need to handle issue statuses from Jira as well. So we have to look at the data, and make sure we can migrate to this structure.